app/contexts/location-and-service/LocationAndService.context.tsx

```diff
@@ -43,7 +48,7 @@ export const LocationAndServiceProvider: FunctionComponent<
     const [locationTextInput, setLocationTextInput] = useState("");
     const [service, setService] = useState("");
 
-    const value = {
+    const props = {
```

Comment:
Great job!! Let's keep this const as `value` since it reflects the values of our object. `props` is a little too generic and hard to tell what we're referring to here and technically we're passing an object of values, not props :)
```suggestion
    const value = {
```

Reply:
Ok, sounds good!